### PR TITLE
Updates to tour story

### LINKED ISSUE #3396: Update Fire tour story to include vectors OR create new fire tour story
Update fire tour story to include:
- Vectors
- Links to FIRMS US/Canada
- PyroCB AI layer?
- New night lights layer and yellow-blue (if you can see fires in it?)
- geocolor geostationary

### PR DESCRIPTION
**Pull request by minniewong**
Imagery mostly pointing to 2021 events, add geocolor, pyrocb, vector and new night light steps.

## Description

Fixes #3396 .

Updated Fire Tour Story to include- vectors, geostationary, blue/yellow night lights, pyroCb aerosol index. Updated much of the imagery to 2021 as well.

[If this is a relatively large or complex change, start a discussion by explaining why you chose the solution you did and what alternatives you considered, etc...]

## How To Test

[Provide whatever information a reviewer might need to know in order to verify that the changes made in this PR are working as expected. If there are special build steps that need to be taken in order to get these changes to run (building while on a separate branch, running `npm ci`, etc) include them here.]

- For bugfixes: What steps need to be taken in the UI to verify the bug is fixed?
- For enhancements and features: What is the expected functionality being added/modified? How can a reviewer verify this?

1. Open with these URL parameters
2. ...
3. Verify expected result

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview

### CODE REVIEW

**Review by jasontk19 — APPROVED** [review]
Looks good to me, just some minor verbiage suggestions.

**Review by edplato — APPROVED** [review]
👍🏽

File: config/default/common/config/metadata/stories/fires_overview_2019/step003.md [hunk 1]
```diff
@@ -1,3 +1,2 @@
...
+When you are zoomed in enough on the map, a blue pointer icon will show next to your fire layer. This indicates that you can click on an individual fire point and retrieve information about that fire/thermal anomaly. Try clicking on an orange dot on the map to see the following information: Latitude, Longitude, Brightness Temperature (Channel 21/22) (for MODIS)/ (Channel I-4) (For VIIRS), Brightness Temperature (Channel 31) (For MODIS)/ (Channel I-5) (for VIIRS), Fire Radiative Power, Detection Confidence, Day/Night Flag, Along-Scan Pixel Size, Along-Track Pixel Size, Acquisition Date, Acquisition Time (in UTC), Satellite, Collection and Source. See detailed descriptions of the attributes for [VIIRS](https://earthdata.nasa.gov/earth-observation-data/near-real-time/firms/v1-vnp14imgt#ed-viirs-375m-attributes) and [MODIS](https://earthdata.nasa.gov/earth-observation-data/near-real-time/firms/c6-mcd14dl#ed-firms-attributes) fire points.
```
[the remark this answers is not in the document]
Comment (minniewong) [reply config/default/common/config/metadata/stories/fires_overview_2019/step003.md:1]: agreed - I was on the fence about whether to have all of that in there as well!

**Review by rboller — COMMENTED** [review]
Step 1
- "Satellite derived" --> "Satellite-derived"?

Step 3
- "Try clicking on an orange doton the map!"

Step 5
- "night time" --> "nighttime" (in two spots)?

Step 8
- will GIBS be saving the demoed GeoColor range indefinitely? 

Step 9
- "Washington State" --> "Washington state" or "State of Washington" according to wikipedia

General:
- consider reducing the number layers in the list on each step to only what's needed since some steps ask users to toggle layers etc and it may take some fishing
- otherwise the updates look great!

**Comment by minniewong** [comment]
@rboller I have made some updates. I have also requested GIBS to preserve that GeoColor time range.

**Review by rboller — APPROVED** [review]
Updates look great to me!

**Pull request merged**